lib/config.yml

```diff
@@ -151,6 +151,9 @@ Metrics/PerceivedComplexity:
 Naming/AccessorMethodName:
   Enabled: false
 
+Naming/BlockForwarding:
+  EnforcedStyle: explicit
```

Comment:
See https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Naming/BlockForwarding for details.

IMO I don't like the overly terse shorthand syntax Ruby has introduced everywhere (programming with symbols everywhere gives me bad memories of C++) so I defaulted to `explicit` here to maintain the existing behavior. But we could just disable this one since I'm sure some people will like it.